README.md

```diff
@@ -157,7 +157,8 @@ to regenerate the client code:
 
 * download the latest API specs from [https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/public_api_v1.yaml]
 * install the [AutoRest](https://github.com/Azure/autorest) tool
-* run the command `autorest --input-file=public_api_v1.yaml --nodejs --output-folder=app/src/api`
+* run the command `autorest --input-file=public_api_v1.yaml --typescript --output-folder=app/src/api --add-credentials`
+* overwrite the contents of `src/api` in this repo with the contents of the folder `app/src/api` 
```

Comment:
is this a manual step?

Reply:
ATM yes, I'll automate this in https://www.pivotaltracker.com/story/show/156568809